snp_motion_planning/src/planning_server.cpp

```diff
@@ -553,11 +554,51 @@ class PlanningServer
     tesseract_planning::TaskComposerFuture::UPtr result = executor->run(*task, problem, task_data);
     result->wait();
 
-    // Save the output dot graph
+    // Save the output motion planning data
     {
-      std::ofstream tc_out_results(tesseract_common::getTempPath() + task_name + "_results.dot");
-      static_cast<const tesseract_planning::TaskComposerGraph&>(*task).dump(tc_out_results, nullptr,
-                                                                            result->context->task_infos.getInfoMap());
+        // Make debug directory
+        std::string time_stamp = tesseract_common::getTimestampString();
+        std::filesystem::path tmp_directory("/tmp");
+        std::filesystem::path snp_parent_dir("SNP_Planning");
+        std::filesystem::path timestamped_dir("Plan_" + time_stamp);
+        std::filesystem::path debug_directory = tmp_directory / snp_parent_dir / timestamped_dir;
+        std::filesystem::create_directories(debug_directory);
+        RCLCPP_INFO_STREAM(node_->get_logger(), "Saving to: " << debug_directory);
+
+        // Add debug trajectories to be plottable
+        std::vector<tesseract_common::JointTrajectory> debug_trajectories;
```

Comment:
This list doesn't seem to be used anywhere (other than to put things into it). We should remove it